source/msoAutoShapeTypes.py

```diff
@@ -0,0 +1,754 @@
+msoShapeSmileyFace=17
+msoShapeSnip1Rectangle=155
+msoShapeSnip2DiagRectangle=157
+msoShapeSnip2SameRectangle=156
+msoShapeSnipRoundRectangle=154
+msoShapeSquareTabs=170
+msoShapeStripedRightArrow=49
+msoShapeSun=23
+msoShapeSwooshArrow=178
+msoShapeTear=160
+msoShapeTrapezoid=3
+msoShapeUpArrow=35
+msoShapeUpArrowCallout=55
+msoShapeUpDownArrow=38
+msoShapeUpDownArrowCallout=58
+msoShapeUpRibbon=97
+msoShapeUTurnArrow=42
+msoShapeVerticalScroll=101
+msoShapeWave=103
+
+msoAutoShapeTypeToRoleText={
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape10pointStar:pgettext("shape","10-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape12pointStar:pgettext("shape","12-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape16pointStar:pgettext("shape","16-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape24pointStar:pgettext("shape","24-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape32pointStar:pgettext("shape","32-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape4pointStar:pgettext("shape","4-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape5pointStar:pgettext("shape","5-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape6pointStar:pgettext("shape","6-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape7pointStar:pgettext("shape","7-point star"),
+	# Translators: a shape name from Microsoft Office"),
+	# See MSOAutoShapeType enumeration from https://msdn.microsoft.com/en-us/library/office/ff862770.aspx?f=255&MSPPError=-2147217396
+	msoShape8pointStar:pgettext("shape","8-point star"),
+	# Translators: a shape name from Microsoft Office"),

```

Comment:
Many of these lines have `")` on the end of them. I think this is a copy paste mistake? Also do we need to repeat the URL, I'm guessing so that it appears in the translator comment for each of these strings. It would be nice if we could specify the URL just once somehow.